qmetaobject/src/qtdeclarative.rs

```diff
@@ -506,11 +506,7 @@ pub fn qml_register_type<T: QObject + Default + Sized>(
 ///
 /// [qt]: https://doc.qt.io/qt-5/qqmlengine.html#qmlRegisterModule
 #[cfg(qt_5_9)]
-pub fn qml_register_module(
-    uri: &CStr,
-    version_major: u32,
-    version_minor: u32,
-) {
+pub fn qml_register_module(uri: &CStr, version_major: u32, version_minor: u32) {
```

Comment:
Rustfmt just won't give up xD

Reply:
Shouldn't we just let rustfmt do its thing? We will probably need to manually change this line every time rustfmt is run otherwise.  I currently use formatted in neovim to run rustfmt automatically on save.

Reply:
I extracted it into separate (first) commit on its own, just to keep history clean and on point :)